src/WarcraftLegacies.Source/Quests/Lordaeron/QuestTyrHand.cs

```diff
@@ -29,19 +30,19 @@ public sealed class QuestTyrHand : QuestData
       "The city of Tyr's Hand is considered impregnable, but they will be reluctant to join the war",
       @"ReplaceableTextures\CommandButtons\BTNHumanBarracks.blp")
     {
-      AddObjective(new ObjectiveEitherOf(
-        new ObjectiveEitherOf(new ObjectiveCapitalDead(capitalCity),new ObjectiveCapitalDead(stratholme)),
-        new ObjectiveCapitalDead(lichKing)));
+      AddObjective(new ObjectiveEitherOf(new ObjectiveCapitalDead(stratholme),
+        new ObjectiveTime(840)));
       AddObjective(new ObjectiveSelfExists());
       _rescueUnits = rescueRect.PrepareUnitsForRescue(RescuePreparationMode.HideNonStructures);
+      ResearchId = Constants.UPGRADE_R023_QUEST_COMPLETED_THE_FORTIFIED_CITY;
       Required = true;
     }
 
     /// <inheritdoc />
     protected override string RewardFlavour => "The city-fortress of Tyr's Hand has decided to join us!";
```

Comment:
From a flavor standpoint - need to think about why Tyr's Hand gives you siege weapons. I think it'd be fine making something up here about them having an abundance of workshops or something, I don't know.